devel/dparser/Portfile

```diff
@@ -19,14 +19,15 @@ long_description \
     just well formed computer languages and data files, but just about any \
     wacky situation that occurs in the real world.
 
-homepage            http://dparser.sourceforge.net/
-master_sites        sourceforge
-checksums           rmd160  5075cede706ee98c4c7bec8288f4ef919bf1dc97 \
-                    sha256  4c4a61bd5954d75da89cbcfe8f6aac3a113769939b31ce2bf4b20b7b3a31cdd8
-distname            d-${version}-src
+homepage            http://github.com/jplevyak/dparser
```

Comment:
Since the source is now on Github, the Github portgroup should be used; it will automate most of the work.

Reply:
FYI, I believe "https:" is the correct URL protocol these days. But that could be fixed in a future update.

Reply:
Oh, and also, it isn't needed at all, what was I thinking, the portgroup handles it already by default. If I commit this now, please submit a PR to fix it later.